src/arch/ArchHooks/ArchHooks_Win32Static.cpp

```diff
@@ -10,59 +10,160 @@
 #include <vector>
 #include <mutex> // for call_once
 
-// for QueryPerformanceCounter
-#include <windows.h>
-#include <mmsystem.h>
+#include <windows.h> // for QueryPerformanceCounter
+#include <mmsystem.h> // for timeBeginPeriod
 #if defined(_MSC_VER)
 #pragma comment(lib, "winmm.lib")
 #endif
 
-static std::once_flag g_timerInitFlag;
-
-/* QueryPerformanceCounter variables below.
- * QueryPerformanceCounter and QueryPerformanceFrequency expect
- * a LARGE_INTEGER, which is a union. These functions store data
- * in the QuadPart of the LARGE_INTEGER, which is a 64-bit integer. */
 namespace {
-    LARGE_INTEGER g_liFrequency;
-    LARGE_INTEGER g_liCurrentTime;
-}  // namespace
+	static std::once_flag g_timer_init_flag;
```

Comment:
Variables with static linkage actually follow the kSnakeCase format instead of snake_case:

https://google.github.io/styleguide/cppguide.html#Constant_Names

I'm okay keeping it at g_timerInitFlag or we rename it to the follow the google convention.